addons/mail/static/src/xml/web_kanban_activity.xml

```diff
@@ -4,7 +4,7 @@
 <t t-name="mail.KanbanActivity">
     <div class="o_kanban_inline_block dropdown o_kanban_selection o_mail_activity">
         <a class="dropdown-toggle o-no-caret o_activity_btn" data-toggle="dropdown" role="button">
-            <span class="fa fa-clock-o fa-lg fa-fw" t-att-title="widget.selection[widget.activityState]" role="img" t-att-aria-label="widget.selection[widget.activity_state]"/>
+            <span t-attf-class="fa-lg fa-fw fa #{widget.recordData.activity_exception_decoration ? 'pull-right text-'+ widget.recordData.activity_exception_decoration +' '+ widget.recordData.activity_exception_icon : 'fa-clock-o'}" t-att-title="widget.selection[widget.activityState]" role="img" t-att-aria-label="widget.selection[widget.activity_state]"/>
```

Comment:
why do you add class here if you already do it in JS (see https://github.com/odoo/odoo/pull/31839/files#diff-75c502c511f53d668928d69e9341d308R692 )

Reply:
@mge-odoo Done